pkg/kubelet/server/stats/types.go

```diff
@@ -64,6 +66,8 @@ type PodStats struct {
 	Containers []ContainerStats `json:"containers" patchStrategy:"merge" patchMergeKey:"name"`
 	// Stats pertaining to network resources.
 	Network *NetworkStats `json:"network,omitempty"`
+	// Stats pertaining to volume usage of filesystem resources.
+	VolumeStats []VolumeStats `json:"volume,omitempty"`
```

Comment:
Though not strictly necessary, can you add: `patchStrategy:"merge" patchMergeKey:"name"` for consistency.